emoji-ui/src/components/emojiItem.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import './emojiItem.css';
+
+class EmojiItem extends Component {
+  static propTypes = {
+    isSelected: PropTypes.bool.isRequired,
+    url: PropTypes.string.isRequired,
+    clickHandler: PropTypes.func.isRequired.isRequired,
+  };
+
+  render() {
+    const { isSelected, url, clickHandler } = this.props;
+    const styles = { backgroundImage: `url(${url})` };
+    const containerClasses = `Emoji-container${isSelected ? ' Emoji-container-selected' : ''}`;
```

Comment:
nit: I think in bem this would be --selected (two dashes) since the "--selected" part is a modifer.

(also, i prefer all standard kebab case instead of mixed upper/lower kebab, but that's style and totally your call.)

Reply:
i can change it to standard all lowercase kebab style. i also prefer it this way. i was following the style in `App.css`.

Reply:
oh, i hadn't even looked at App.css yet. :D